src/runtime_action.rs

```diff
@@ -155,13 +155,25 @@ impl RuntimeAction {
         match self {
             RuntimeAction::RateLimit(rl_action) => {
                 let descriptor = rl_action.build_descriptor()?;
-                if descriptor.entries.is_empty() {
+                let known_attrs = rl_action.get_known_attributes().unwrap_or_default();
+
+                if descriptor.entries.is_empty() && known_attrs.is_empty() {
```

Comment:
I don't think we want to do the call when `known_attrs` are there... afaik we _need_ at least one descriptor entry for limitador to be able to discriminate which `Limit`s to apply

Reply:
Right, the following action should not trigger gRPC call. 

```yaml
actions:
- service: ratelimit-service
  scope: ratelimit-scope-a
  data:
    - expression:
        key: ratelimit.domain
        value: foobar
    - expression:
        key: ratelimit.hits_addend
        value: '3'
```